R/utils.R

```diff
@@ -301,16 +301,19 @@ file_created_dance <- function(
   dir, 
   name, 
   open_file, 
+  open_or_go_to = TRUE,
   catfun = cat_created
 ){
   catfun(where)
   
   fun(pkg, dir, name)
   
-  open_or_go_to(
-    where = where,
-    open_file = open_file
-  )
+  if (open_or_go_to){
+      open_or_go_to(
+      where = where,
+      open_file = open_file
+    )
+  }
```

Comment:
Thanks a lot @novica !! 

I have a last request: given that `open_or_go_to()` returns `invisible(where)`, we would miss that return value if `open_or_go_to = FALSE` here. 

Can you add an else that `return(invisible(where))` please?

Reply:
Done.